recipes/watchdog-wms/build.sh

```diff
@@ -19,12 +19,16 @@ mv "${OUT}/workflowDesigner.sh" "${OUT}/watchdog-gui.sh"
 ln -s "${OUT}/watchdog-cmd.sh" "${PREFIX}/bin/watchdog-cmd"
 chmod 0755 "${PREFIX}/bin/watchdog-cmd"
 
+# get the modules
+/bin/bash "${OUT}/modules/downloadCommunityModules.sh" --auto
```

Comment:
What's this downloading?

Reply:
Modules for the workflow management system that were previously shipped together with Watchdog but are now located at https://github.com/watchdog-wms/watchdog-wms-modules.

Reply:
We really try not to vendor so many packages, why not use conda to install most of those?

Reply:
In almost all cases the modules only contain specifications on how a binary could be used (XSD files). The dependend binaries must be installed on the system and are not part of that repository.
But probably it might be better to leave that out because the modules are then up-to date after installation.